.github/workflows/ersem.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: build-ersem
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-fabm-ersem:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout branch
+          uses: actions/checkout@v2
+        - name: Setup python
+          uses: actions/setup-python@v2
+          with:
+              python-version: '3.x'
+        - name: Installing wheel and numpy
+          run: python -m pip install wheel numpy
+        - name: Cloning FABM
+          run: git clone https://github.com/fabm-model/fabm.git
+        - name: Building FABM-ERSEM
+          run: |
+              mkdir build && cd build
+              cmake ../fabm/src/drivers/python -DFABM_ERSEM_BASE=..
+              make install
+        - name: Check installation
+          run: python -c "import fabm"
+  build-gotm-fabm-ersem:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout branch
+          uses: actions/checkout@v2
+        - name: Setup python
+          uses: actions/setup-python@v2
+          with:
+              python-version: '3.x'
+        - name: Installing gfortran and netCDF
+          run: sudo apt-get install gfortran libnetcdff-dev
+        - name: Cloning FABM and GOTM
+          run: |
+              git clone https://github.com/fabm-model/fabm.git
+              git clone https://github.com/gotm-model/code.git gotm
+              cd gotm && git submodule update --init --recursive && cd ..
+        - name: Building GOTM-FABM-ERSEM
+          run: |
+              mkdir build && cd build
+              cmake ../gotm/src -DFABM_BASE=../fabm -DFABM_ERSEM_BASE=.. -DCMAKE_Fortran_COMPILER=/usr/bin/gfortran
```

Comment:
Better change `../gotm/src` to `../gotm`? The former is deprecated and will break someday. There is a warning somewhere among all the cmake output.

Reply:
Corrected